doc/nrf-bm/includes/create_sample.txt

```diff
@@ -3,14 +3,20 @@ Creating a copy of the sample
 
 Follow these steps to create a new application based on this sample:
 
-1. Open the nRF Connect for VS Code extension by clicking its icon.
+1. Open the |nRFVSC| by clicking its icon.
 #. From the :guilabel:`Welcome View`, click :guilabel:`Create a new application`.
 
    A quick pick menu appears.
 
 #. Select :guilabel:`Copy a sample` - this will create a `freestanding application`_ from a sample in |BMlong|.
 
+   A quick pick menu appears showing a list of SDK versions to copy the sample from.
+
+#. Select the SDK version that you want to copy the sample from.
```

Comment:
I see that this SDK version quick pick menu only appear if you have more than one SDK installed.
Users that install for the first time will most likely not see this.

Maybe something like this?
```suggestion
   If you have more than one SDK version installed, a quick pick menu appears showing a list of SDK versions to copy the sample from.

#. If asked to select the SDK version, select the SDK version that you want to copy the sample from.
```